.github/workflows/core.yml

```diff
@@ -1,11 +1,14 @@
 name: core
+
 on:
   push:
+    branches:
+      - master
+      - branch-*
   pull_request:
     branches:
       - master
       - branch-*
-    types: [opened, synchronize]
```

Comment:
> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request